slave-dotnet20/Dockerfile.rhel7

```diff
@@ -0,0 +1,37 @@
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet20-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet20-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Slave .NET Core 2.0" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
```

Comment:
Suggested wording: "The .NET Core Jenkins Slave image contains dotnet tools on top of the Jenkins slave base image."